encrypted-media/scripts/requestmediakeysystemaccess.js

```diff
@@ -0,0 +1,311 @@
+function runTest( config, qualifier ) {
+
+    var testname = testnamePrefix( qualifier, config.keysystem ) + ', requestMediaKeySystemAccess';
+
+    function expect_error(keySystem, configurations, expectedError, testname) {
+        promise_test(function(test) {
+            return navigator.requestMediaKeySystemAccess(keySystem, configurations).then(function(a) {
+                assert_unreached('Unexpected requestMediaKeySystemAccess() success.');
+            }, function(e) {
+                assert_equals(e.name, expectedError);
+            });
+        }, testname);
+    }
+
+    function assert_subset(actual, expected, path) {
+        if (typeof expected == 'string') {
+            assert_equals(actual, expected, path);
+        } else {
+            if (expected.hasOwnProperty('length'))
+                assert_equals(actual.length, expected.length, path + '.length');
+            for (property in expected)
+                assert_subset(actual[property], expected[property], path + '.' + property);
+        }
+    }
+
+    function expect_config(keySystem, configurations, expectedConfiguration, testname) {
+        promise_test(function(test) {
+            return navigator.requestMediaKeySystemAccess(keySystem, configurations).then(function(a) {
+                assert_subset(a.getConfiguration(), expectedConfiguration, 'getConfiguration()');
```

Comment:
For any configuration the description will start with 'getConfiguration". Can we make it more specific?